starter-kits.json

```diff
@@ -2,11 +2,15 @@
   "angular-apollo-tailwind": "Angular, Apollo, and TailwindCSS",
   "cra-rxjs-styled-components": "Create React App, RxJS and Styled Components",
   "next12-react-query-tailwind": "NextJS 12, React Query, and TailwindCSS",
+  "next12-routing-chakra-ui": "NextJS 12 and Chakra UI",
   "remix-gql-tailwind": "Remix, GQL and TailwindCSS",
   "nuxt2-pinia-tailwind": "Nuxt 2, Pinia and TailwindCSS",
   "svelte-kit-scss": "SvelteKit and SCSS",
   "vue3-apollo-quasar": "Vue3, Apollo, and Quasar",
   "qwik-graphql-tailwind": "Qwik, GraphQL, and TailwindCSS",
+  "solidjs-tailwind": "SolidJs and TailwindCSS",
+  "next12-routing-chakra-ui": "NextJS 12 and Chakra UI",
+  "angular-ngrx-scss": "Angular, NgRx, and SCSS"
```

Comment:
Are these other kits showing as additions because the branch is behind main?

Reply:
yes